mimic/model/glance_objects.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
Yea, that makes sense.  One thing to consider here is that only the IAD endpoint returns onMetal servers. Also we need a way to update these images when someone creates an image of an existing server since saved images come back in the images call of compute and glance